core/src/main/scala/cromwell/core/ExecutionStatus.scala

```diff
@@ -3,10 +3,11 @@ package cromwell.core
 object ExecutionStatus extends Enumeration {
   type ExecutionStatus = Value
   val NotStarted, Starting, Running, Failed, Preempted, Done, Aborted = Value
+  val TerminalStatuses = Set(Failed, Done, Aborted, Preempted)
```

Comment:
I agree, although I don't want to pollute this PR with completely unrelated changes. I'm happy to change this in a subsequent PR though.